configs/configs.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configs
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"time"
+)
+
+// JSONDuration is an abstraction used to simplifying unmarshalling
+// json into structs that contain time.Duration fields. Expected
+// format is the same as provided to time.ParseDuration.
+type JSONDuration struct {
+	time.Duration
+}
+
+// UnmarshalJSON parses a time.Duration field from a string
+func (jd *JSONDuration) UnmarshalJSON(data []byte) error {
+	s := ""
+	err := json.Unmarshal(data, &s)
```

Comment:
nit: Scope err in the if? `if err := json.Unmarshal(data, &s); err != nil { ... }`